data/remnant/remnant missions.txt

```diff
@@ -3455,3 +3455,264 @@ event "remnant: pantica void sprites"
+		outfit "Research Laboratory"
+		conversation
+			`You follow the directions of the starport control and settle onto a pad next to the outfitter. You barely have a chance to finish your landing checks before a team of Remnant have a compact pod waiting on your loading ramp. Once you open the cargo bay doors they pick it up and maneuver it inside. They start securing it to the ship while one approaches you and Dawn.`
+			`	"We have loaded the research laboratory onboard. The team will install it if they can find the space, but if they do not, it will be secured in the cargo bay." The Remnant turns to you as she continues. "Before you depart, please verify that the laboratory is installed in the correct location. You know your ship better that we do, after all." She turns back to Dawn and makes a gesture of respect. "Embers guide your path, Seeker." With that, she strides away into the hive of activity that is the outfitting bays.`
+			choice
+				`	"So, what is our next step?"`
+					goto labnext
+				`	"Seeker?" you ask.`
+			`	Dawn looks a little embarassed. "Seeker is a title of respect for deep space explorers. It is unofficial - just a token of respect - but it is both a title that I aspire to earn and one I do not feel that I deserve."`
+			choice
+				`	"Okay. So what next?"`
+					goto labnext
+				`	"I thought all Remnant were spacefarers?"`
+			`	"Oh, we are. Every Remnant has had some experience in space. But there is a big difference between a rotation of a month or two of duty in a spacefaring post and spending months or years drifting in the void surrounding distant systems."`
+			choice
+				`	"Okay, let's get on with this."`
+					goto labnext
+				`	"Years drifting in the void?"`
+			`	"If you want good data, you need to spend time to get it. And especially when we are tracking stellar phenomenon, you need periods of long observation. Snapshots of the sky are good - that is what we will be collecting on this trip. But for watching the galaxy at large, other civilizations, and things of that ilk... Nothing is better than long-term observation." She looks lost in thought for a moment.`
+			label labnext
+			`	"That is neither here nor there, however. Now that you have the research laboratory, you will need to locate a jump drive in order to reach Sagittarius A*." She pauses and checks her commlink. "If you do not already have one, you can salvage one from a Korath Raider. Our early warning system indicates that there are several incoming Korath fleets that should be here in the next day or two. Any one of them should have a Jump Drive you can take. Meet me in the Starport once you have a Jump Drive."`
+
+
+
+event "remnant: lab availability"
+	outfitter "Remnant"
+		add "Research Laboratory"
+
+
+
+mission "Remnant: Expanded Horizons Astral 2"
+	name "Research expedition to Sagittarius A*"
+	description "Travel to Sagittarius A* and then return to <destination>."
+	blocked "You need at least 1 free bunk to take this mission."
```

Comment:
This could also be blocked by not having a JD or lab.

Reply:
Fixed. Thanks!